docs/USAGE-GUIDE.md

```diff
@@ -77,6 +77,31 @@ And when invoking CMake for your project, do one of two things:
  1. Set the `CMAKE_INSTALL_PREFIX` variable with the path to your s2n-tls build.
  2. If you have globally installed s2n-tls, do nothing, it will automatically be found.
 
+
+### 32 Bit Platforms
+s2n-tls does not support 32 bit builds. Our CI does not build or test on 32 bit platforms, so they may or may not work. If you run into a problem with 32 bit builds you can open an issue and we may or may not be able to help you. Note that we are unlikely to accept PR's for 32 bit issues at this time since we cannot properly test them. We hope to offer improved 32 bit support in the future, contingent upon updating our CI with coverage for 32 bit platforms.
```

Comment:
```suggestion
s2n-tls does not support 32 bit builds. Our CI does not build or test on 32 bit platforms, so they may or may not work. If you run into a problem with 32 bit builds you can open an issue and we may or may not be able to help you. Note that we are unlikely to accept PRs for 32 bit issues at this time since we cannot properly test them. We hope to offer improved 32 bit support in the future, contingent upon updating our CI with coverage for 32 bit platforms.
```